src/views/Curators/Curator.vue

```diff
@@ -412,17 +310,26 @@
             requests.forEach(item => {
               let object = {};
               object.createdAt = item.createdAt;
-              object.recordNameID = item.fairsharingRecord.name+' ('+item.fairsharingRecord.id+')';
-              object.userNameID = item.user.username+' ('+item.user.id+')';
+              object.recordName = item.fairsharingRecord.name + ' ('+ item.fairsharingRecord.id +')';
+              object.id = item.fairsharingRecord.id;
+              object.type = item.fairsharingRecord.type ;
+              object.userNameID = item.user.username + ' ('+item.user.id+')';
+              object.processingNotes = item.fairsharingRecord.processingNotes;
+              object.requestID = item.id;
```

Comment:
Single object with string interpolation:
```suggestion
              let object = {
                createdAt: item.createdAt,
                recordName: `${item.fairsharingRecord.name} (${item.fairsharingRecord.id)})`,
                id: item.fairsharingRecord.id,
                type: item.fairsharingRecord.type,
                userNameID: `${item.user.username} (${item.user.id})`,
                processingNotes: item.fairsharingRecord.processingNotes,
                requestID: item.id
            };
```

Reply:
In the last commit, I changed manually all the object declaration to init the fields there as comment suggested.